frontend/src/components/user/builder/builder_signup.vue

```diff
@@ -317,7 +334,7 @@
             // done any other actions to refresh user information
             this.$store.commit('set_current_user', response.data.user)
 
-            this.route_set_password()
+            this.route_new_project()
```

Comment:
The builder sign up is the next step right?

Reply:
No, this is already builder signup

Reply:
oh ok I got order mixed up I think thanks